orchagent/orch.cpp

```diff
@@ -133,6 +140,48 @@ void Consumer::drain()
         m_orch->doTask(*this);
 }
 
+// TODO: Table should be const
+void Orch::addExistingData(Table *table)
+{
+    string tableName = table->getTableName();
+    Consumer* consumer;
+    auto it = m_consumerMap.begin();
+
+    while (it != m_consumerMap.end())
+    {
+        consumer = (Consumer*)(it->second.get());
```

Comment:
The change overlap with part of #547.  One of the problem here is that other types of class may be in m_consumerMap too and they don't  have getTableName() methods.  casting them to Consumer may cause problem. 
The other reason to add getName()/setName() for Executor is to facilitate debugging.